custodia.spec

```diff
@@ -191,6 +192,24 @@ cp %{buildroot}/%{_bindir}/custodia-cli %{buildroot}/%{_bindir}/custodia-cli-3
 %endif
 
 
+%pre
+getent group custodia >/dev/null || groupadd -r custodia
+if ! getent passwd custodia >/dev/null; then
+     useradd -r -g custodia -d / -s /sbin/nologin -c "User for custodia" custodia
+     if [ -d %{_sysconfdir}/custodia ]; then
+         chown -R custodia:custodia %{_sysconfdir}/custodia
+     fi
```

Comment:
In any case I think we should move user creation etc on it's own PR, this change is important enough to be reviewed separately from other spec fixes.